### PR TITLE
Some modifications to the example files.

### PR DESCRIPTION
**Pull request by adamtopaz**
Adds a few more details in docstrings, a few more examples, and uses `/-!` whenever possible.

### CODE REVIEW

File: src/examples/Ext.lean [hunk 1]
```diff
@@ -4,65 +4,122 @@ import for_mathlib.universal_delta_functor.Ext
 
 /-!
 
-This file shows that `Ext` is a universal δ-functor
-and performs the computation `Ext^1(ℤ/nℤ, ℤ/nℤ) = ℤ/nℤ`.
+This file discussed the various Ext groups appearing in our project.
```
Comment (jcommelin) [inline src/examples/Ext.lean:7]:
```suggestion
This file discusses the various Ext groups appearing in our project.
```

File: src/examples/Ext.lean [hunk 2]
```diff
@@ -4,65 +4,122 @@ import for_mathlib.universal_delta_functor.Ext
...
 -/
 example (n : ℕ) (X Y : 𝓐) :
   Ext' n (op X) Y =
-  Ext n (op ↑X) ↑Y := rfl
+  Ext n (op ↑X) ↑Y :=
+rfl
 
-/-- `Ext' 0 (-, B) ≅ Hom(-,B)` -/
+/-!
+The `Ext' n` can be assembeled into a δ-functor, which is denoted `Ext_δ_functor 𝓐 Y`.
```
Comment (jcommelin) [inline src/examples/Ext.lean:77]:
```suggestion
The `Ext' n` can be assembled into a δ-functor, which is denoted `Ext_δ_functor 𝓐 Y`.
```

File: src/examples/Ext.lean [hunk 3]
```diff
@@ -4,65 +4,122 @@ import for_mathlib.universal_delta_functor.Ext
...
+/-!
+`Ext' 0 (X, Y) ≅ Hom(X,Y)`.
+-/
 example (X Y : 𝓐) : Ext' 0 (op X) Y ≅ AddCommGroup.of (X ⟶ Y) :=
 (Ext'_zero_flip_iso _ _).app _
 
-/-- Any natural transformation `Hom(-,B) ⟶ F 0` to the zeroth-component of some
+/-!
+The isomorphism above is functorial in the first variable, and the isomorphism of functors
+is dentoed `Ext'_zero_flip_iso 𝓐 Y`. This isomorphism will be used in the example below.
```
Comment (jcommelin) [inline src/examples/Ext.lean:99]:
```suggestion
is denoted `Ext'_zero_flip_iso 𝓐 Y`. This isomorphism will be used in the example below.
```

File: src/examples/pBanach.lean [hunk 4]
```diff
@@ -17,8 +17,7 @@ variables (p : ℝ≥0) (V : pBanach.{0} p) (S : Profinite.{0})
 
 section pBanach
 
-/-! The $p$-Banach space $V$ is a complete and separated topological $ℝ$-vector space. -/
-
+/-! The $p$-Banach space $V$ is a complete and separated topological `ℝ`-vector space. -/
```
Comment (jcommelin) [inline src/examples/pBanach.lean:20]: This is now mixing `$` and `` ` ``. Is that intentional?

File: src/examples/radon_measures.lean [hunk 5]
```diff
@@ -13,20 +13,21 @@ open liquid_tensor_experiment category_theory category_theory.limits
 
 variables (p : ℝ≥0) [fact (0 < p)] [fact (p ≤ 1)]
 
-/-
+/-!
 The functor which associates a condensed abelian group to a
-CompHaus-ly filtered pseudo normed group.
+CompHaus-ly filtered pseudo normed group is denoted by
+`CompHausFiltPseuNormGrp.to_Codnensed`.
```
Comment (jcommelin) [inline src/examples/radon_measures.lean:19]:
```suggestion
`CompHausFiltPseuNormGrp.to_Condensed`.
```

**Review by jcommelin — APPROVED** [review]

File: src/examples/pBanach.lean (hunk 4 above)
Comment (adamtopaz) [reply src/examples/pBanach.lean]: I changed all the `$` to `\`` in this file.

**Pull request merged**